verifier/src/tcb.rs

```diff
@@ -388,35 +518,81 @@ mod tests {
                       "svn": 4,
                       "category": "It's a category with a type",
                       "type": "It's a type to go with the category"
+                    },
+                    {
+                      "svn": 5
```

Comment:
With the move from a vec to an array this grew a bit bigger than ideal.
I'm leaning toward removing this tests since the `example_tcb.json` contains these optional fields and that's probably enough to test the correct behavior.

Reply:
I think it's okay, there's no need to make the tests perfect -- if we have the tests and they are passing no need to futz with it more